project-templates/node-vscode-json/glsp-client/tasklist-vscode/extension/package.json

```diff
@@ -0,0 +1,162 @@
+{
```

Comment:
I think the webview.js (above, can't add a comment there directly) shouldn't be committed right? Isn't it generated by the build?

Reply:
Good catch.  Has been removed and added to .gitignore